src/index.html

```diff
@@ -4,9 +4,410 @@
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <div class="page-wrapper">
+      <header class="header">
+        <a href="#home" class="header__link">
+          <img 
+            src="./images/logo.png" 
+            alt="Moyo-logo" 
+            class="header__logo"
+          >
+        </a>
+  
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#" class="nav__link is-active">Apple</a>
+            </li>
+  
+            <li class="nav__item">
+              <a href="#" class="nav__link">Samsung</a>
+            </li>
+  
+            <li class="nav__item">
+              <a href="#" class="nav__link">Smartphones</a>
+            </li>
+  
+            <li class="nav__item">
+              <a 
+                href="#" 
+                class="nav__link" 
+                data-qa="nav-hover"
+              >
+                Laptops & Computers
+              </a>
+            </li>
+  
+            <li class="nav__item">
+              <a href="#" class="nav__link">Gadgets</a>
+            </li>
+  
+            <li class="nav__item">
+              <a href="#" class="nav__link">Tablets</a>
+            </li>
+  
+            <li class="nav__item">
+              <a href="#" class="nav__link">Photo</a>
+            </li>
+  
+            <li class="nav__item">
+              <a href="#" class="nav__link">Video</a>
+            </li>
+          </ul>
+        </nav>
+      </header>
+  
+      <main class="main">
+        <section class="product-card" data-qa="card">
+          <img 
+            src="./images/imac.jpeg" 
+            alt="APPLE A1419 iMac 27" 
+            class="product-card__image"
+          >
+  
+          <h2 class="product-card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+  
+          <p class="product-card__subtitle">Product code: 195434</p>
+  
+          <div class="rates">
+            <div class="rates__stars">
+              <div class="rates__star"></div>
+  
+              <div class="rates__star"></div>
+  
+              <div class="rates__star"></div>
+  
+              <div class="rates__star"></div>
+          
+              <div class="rates__star"></div>
+            </div>
+  
+  
+            <div class="rates__reviews">Reviews: 5</div>
+          </div>
+  
+          <div class="product-card__price-block">
+            <div class="product-card__price-subtitle">Price:</div>
+  
+            <div class="product-card__price">$2,199</div>
+          </div>
+          
+          <a 
+            href="#" 
+            class="product-card__buy-link" 
+            data-qa="card-hover"
+          >
+            Buy
+          </a>
+  
+        </section>
```

Comment:
It is necessary to don't put empty space between parent element and child element